src/ShopifyApp/Traits/AuthControllerTrait.php

```diff
@@ -207,4 +185,28 @@ protected function returnTo()
         // No return_to, go to home route
         return redirect()->route('home');
     }
+
+    /**
+     * Fires ש job.
+     *
+     * @param array $config The job's configuration
+     *
+     * @return bool
+     */
+    protected function fireJob($job)
```

Comment:
I believe this will fail since $job needs to be $config. Did you update the tests as well?

Reply:
Not sure I understand your comment, I simply changed some structure, didn't change any logic. I moved the function to be a class function instead of an anonymous function inside the afterAuthenticateJob function.